src/flynt/cli.py

```diff
@@ -96,11 +96,25 @@ def main():
     )
 
     parser.add_argument(
-        "src", action="store", nargs="+", help="source file(s) or directory"
+        "src", action="store", nargs="*", help="source file(s) or directory"
     )
 
+    parser.add_argument(
+        "--version",
+        action="store_true",
+        default=False,
+        help="Print the current version number and exit."
+    )
     args = parser.parse_args()
 
+    if args.version:
+        print(__version__)
+        sys.exit(0)
+    elif not args.src:
+        print("flynt: error: the following arguments are required: src")
+        sys.exit(1)
+
+    print(f"Running flynt v.{__version__}")
```

Comment:
as noted by @cphyc in private, an alternative to this workaround would be to write

```python
    action_group = parser.add_mutually_exclusive_group()

    action_group.add_argument(
        "src", action="store", nargs="*", help="source file(s) or directory",
        default=["."]
    )

    action_group.add_argument(
        "--version",
        action="store_true",
        default=False,
        help="Print the current version number and exit."
    )
    args = parser.parse_args()

    if args.version:
        print(__version__)
        sys.exit(0)
```

with the notable side-effect of giving `src` a default value. Depending on wether this side effect is desirable or not, this might be the better solution here.